### PR TITLE
Add logs to graphic demo

### PR DESCRIPTION
**Pull request by jacklehamster**

### CODE REVIEW

File: scripts/graphics/index.html [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+        transition: margin-top 0.2s;
+        position: absolute;
+        font-size: 8px;
+    }
+    .logger.shifted {
+        margin-top: 8px;
+        transition: margin-top 0s;
+    }
+</style>
+
```
Comment (Wonay) [inline scripts/graphics/index.html]: Can you put the CSS into the style folder.

File: scripts/graphics/index.html [hunk 2]
```diff
@@ -11,9 +25,27 @@
...
+    });
 
     var canvasViewer = new CanvasViewer(canvas, model);
     canvasViewer.start();
+
+
+    function logAction(action) {
+        var log = document.createElement('div');
+        log.innerText = JSON.stringify(action);
+        var logger = document.getElementById("log");
```
Comment (Wonay) [inline scripts/graphics/index.html]: if you refactor the JS file to be using RequireJS , you will be able to use Jquery for those DOM actions.

File: scripts/graphics/index.html (hunk 1 above)
Comment (jacklehamster) [inline scripts/graphics/index.html]: Done

File: scripts/graphics/index.html (hunk 2 above)
Comment (jacklehamster) [inline scripts/graphics/index.html]: I have no idea how that works

Comment (Wonay) [inline scripts/graphics/index.html]: Look at the other files for examples

File: style/test.css [hunk 3]
```diff
@@ -14,3 +14,14 @@
     padding-left: 32px;
     padding-right: 32px;
 }
+
+.logger {
```
Comment (Wonay) [inline style/test.css]: This file is for the test page. If you create a new page/URL you should create its own css file so its easier to maintain.

**Pull request merged**